viewer/index.html

```diff
@@ -1,84 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>A cheap and cheerful image viewer</title>
-    <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
-    <style>
-      html, body, textarea {
-          width: 100%;
-          height: 100vh;
-          padding: 0;
-          margin: 0;
-      }
-      #error {
-          display: none;
-          background: red;
-          color: white;
-          padding: 20px;
-          text-align: center;
-      }
-      #viewer {
-          display: none;
-          text-align: center;
-      }
-      img {
-          width: 100%;
-          height: 100%;
-          object-fit: contain;
-      }
-    </style>
-  </head>
-  <body>
-    <div id="error"></div>
-    <img id="viewer" />
-    <script>
-      var viewer = document.getElementById('viewer');
-      function showError(msg) {
-        var el = document.getElementById('error')
-        if (!msg) {
-          el.style.display = 'none';
-        } else {
-          el.innerHTML = msg;
-          el.style.display = 'block';
-        }
-      }
-      grist.ready({
-        columns: [{ name: "ImageUrl", title: 'Image URL', type: 'Text'}],
-        requiredAccess: 'read table'
-      });
-      // Helper function that reads first value from a table with a single column.
-      function singleColumn(record) {
-        const columns = Object.keys(record || {}).filter(k => k !== 'id');
-        return columns.length === 1 ? record[columns[0]] : undefined;
-      }
-      grist.onNewRecord(() => {
-        showError("");
-        viewer.style.display = 'none';
-      });
-      grist.onRecord(function(record) {
-        // If user picked all columns, this helper function will return a mapped record.
-        const mapped = grist.mapColumnNames(record);
-        // We will fallback to reading a value from a single column to
-        // support old way of mapping (exposing only a single column).
-        // New widgets should only check if mapped object is truthy.
-        const data = mapped ? mapped.ImageUrl : singleColumn(record);
-        delete record.id;
-        var keys = Object.keys(record);
-        if (data === undefined) {
-          showError("Please choose a column to show in the Creator Panel.");
-        } else {
-          showError("");
-          if (!data) {
-            viewer.style.display = 'none';
-          } else {
-            var parts = data.split(' ');
-            var url = parts[parts.length - 1];
-            viewer.src = url;
-            viewer.style.display = 'block';
-          }
-        }
-      });
-    </script>
-  </body>
+
+<head>
+  <meta charset="utf-8">
+  <title>A cheap and cheerful image viewer</title>
+  <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
+  <style>
+    .icon-arrow-left {
+      mask-image: var(--icon-ArrowLeft);
```

Comment:
I think that adding a hover cursor, to indicate that this is clickable element, would be useful.